Frontend/ui-common/src/Screens/DriverProfileScreen/Controller.purs

```diff
@@ -43,21 +43,24 @@ eval (GetDriverProfileAPIResponseAction (DriverProfileRes resp)) state = do
       updatedState = state{data = getDriverProfile response}
   _ <- pure $ EHU.toggleLoader false
   continueWithCmd updatedState [do 
-    void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = getImage updatedState, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2}
+    let image = getImage updatedState
+    if (not (null image)) then void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = image, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2} else pure unit
     pure NoAction
   ]
 
 eval (PrevImg) state = do 
   let updatedState = state{data{ imgIdx = if state.data.imgIdx /= 0 then state.data.imgIdx - 1 else 0 }}
   continueWithCmd updatedState [do 
-    void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = getImage updatedState, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2}
+    let image = getImage updatedState
+    if (not (null image)) then void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = image, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2} else pure unit
     pure NoAction
   ]
 
 eval (NextImg) state = do 
   let updatedState = state{data{ imgIdx = if DA.length state.data.displayImages - 1 /= state.data.imgIdx then state.data.imgIdx + 1 else state.data.imgIdx }}
   continueWithCmd updatedState [do 
-    void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = getImage updatedState, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2}
+    let image = getImage updatedState
+    if (not (null image)) then void $ runEffectFn1 JB.displayBase64Image JB.displayBase64ImageConfig {source = image, id = getNewIDWithTag "add_image_component_images" , scaleType =  "FIT_CENTER", adjustViewBounds = false, inSampleSize = 2} else pure unit
```

Comment:
remove unnecessary brackets, also move this to a function instead of repeating it thrice